### DISCUSSION TITLE
How to check if there is an active session or not?

### DISCUSSION THREAD

**Discussion by AkiAkshay** [appwrite/appwrite#5034] [Q&A] [answered]
Hi,

Like firebase we can check if there is an active session by FirebaseUser. How can we check on start of my MainActivity that if a user is logged in or not. I am working with java, i have not found any proper solution. Appwrite says to use get() but i am not able to understand how to use it. Any help

**Comment by stnguyen90** [comment, marked as answer]
Please see https://github.com/appwrite/appwrite/discussions/3938#discussioncomment-3746725.
